prow/jobs/kyma/components/etcd-backup-job/etcd-backup-job.yaml

```diff
@@ -4,6 +4,25 @@ test_infra_ref: &test_infra_ref
   path_alias: github.com/kyma-project/test-infra
 
 job_template: &job_template
+  skip_report: false
```

Comment:
Why do we still need `job_template` with golang buildpack version `v20181119-afd3fbd`?
You use new buildpack in releases branches but not in master branch, why? Why does `postsubmits` use old buildpack?

Reply:
I will add comment